src/components/buttons/ArchiveBtn.tsx

```diff
@@ -0,0 +1,28 @@
+'use client'
+
+import { DustbinIcon } from '@/icons'
+import { Stack, Typography } from '@mui/material'
+
+export const ArchiveBtn = ({ isArchived, handleClick }: { isArchived: boolean; handleClick: () => void }) => {
+  return (
+    <Stack
+      direction="row"
+      alignItems="center"
+      columnGap="6px"
+      sx={{
+        padding: '3px 8px',
+        ':hover': {
+          cursor: 'pointer',
+          border: ' 1px solid var(--Black-Alpha-1, rgba(0, 0, 0, 0.12))',
```

Comment:
> var(--Black-Alpha-1

Is this defined in our code? I don't see global CSS variables... I think we should be using theme here?